.buildkite/publish.sh

```diff
@@ -18,7 +18,7 @@ echo "--- yarn version-bump"
 yarn version-bump
 
 echo "--- git push"
-git push https://${GITHUB_TOKEN}@github.com/${REPO_ORG}/${REPO_NAME}.git --follow-tags origin HEAD:master --no-verify
+git push https://${GITHUB_TOKEN}@github.com/${REPO_ORG}/${REPO_NAME}.git --follow-tags origin/master HEAD:master --no-verify
```

Comment:
I think this is not quite right.
Try instead:
```suggestion
git push https://${GITHUB_TOKEN}@github.com/${REPO_ORG}/${REPO_NAME}.git --follow-tags origin refs/heads/master:refs/heads/master --no-verify
```

Reply:
Just took another look and I think it's because I shouldn't be specifying `origin` when I'm already providing the URL. And I think the left hand side should be `HEAD:master` or `HEAD:refs/heads/master` since the buildkite agent checkouts to a specific commit before creating a new commit. WDYT?

Reply:
Cool, yep, give it a try :smile: